web/sites/default/config/views.view.db_search_group_documents.yml

```diff
@@ -444,3 +447,280 @@ display:
+              policy_author: '0'
+              policy_editor: '0'
+              attorney: '0'
+              council_author: '0'
+              council_clerk: '0'
+              budget: '0'
+              park_editor: '0'
+              code_editor: '0'
+              group_creator: '0'
+              alert_editor: '0'
+              sitewide_editor: '0'
+              administrator: '0'
+            placeholder: ''
+          is_grouped: false
+          group_info:
+            label: ''
+            description: ''
+            identifier: ''
+            optional: true
+            widget: select
+            multiple: false
+            remember: false
+            default_group: All
+            default_group_multiple: {  }
+            group_items: {  }
+          parse_mode: terms
+          min_length: null
+          fields:
+            - field_summary_1
+            - name
+            - rendered_item
+          plugin_id: search_api_fulltext
+      filter_groups:
+        operator: AND
+        groups:
+          1: AND
+      arguments:
+        media_display_group_id:
+          id: media_display_group_id
+          table: search_api_index_database_full_index
+          field: media_display_group_id
+          relationship: none
+          group_type: group
+          admin_label: ''
+          default_action: default
+          exception:
+            value: all
+            title_enable: false
+            title: All
+          title_enable: false
+          title: ''
+          default_argument_type: fixed
+          default_argument_options:
+            argument: '232'
+          default_argument_skip_url: false
+          summary_options: {  }
+          summary:
+            sort_order: asc
+            number_of_records: 0
+            format: default_summary
+          specify_validation: false
+          validate:
+            type: none
+            fail: 'not found'
+          validate_options: {  }
+          break_phrase: false
+          not: false
+          plugin_id: search_api
+      header:
+        area_text_custom:
+          id: area_text_custom
+          table: views
+          field: area_text_custom
+          relationship: none
+          group_type: group
+          admin_label: ''
+          empty: false
+          tokenize: false
+          content: "<p>You may submit comments about a liquor license application within 30 days of the application’s receipt by the City. They will become part of the permanent public record for that application, available upon request.</p>\r\n\r\n<p>All comments must include the address of the liquor license application as well as your own name and address, or they will not be considered. To submit comments, email them to <a href=\"mailto:liquor@portlandoregon.gov\">liquor@portlandoregon.gov</a>; or send them by postal mail to:</p>\r\n\r\n<p class=\"mx-auto w-50\">Liquor Licensing<br />\r\n1120 SW 5th Ave, Suite 114<br />\r\nPortland, OR 97204</p>"
```

Comment:
@RichardDavies , we should probably use the address element and let it left justify for the mailing address. I can't think of another place where we center content in this fashion.

Reply:
I've made this change. I didn't realize the HTML 5 spec now allows the address element to be used for general purpose addresses (it used to be just for the document author's contact info.)